includes/Modules/Analytics.php

```diff
@@ -340,9 +348,11 @@ public function get_debug_fields() {
 	 * @param string $property_id Analytics property ID to use in the snippet.
 	 */
 	protected function enqueue_gtag_js( $property_id ) {
+		$gtag_src = "https://www.googletagmanager.com/gtag/js?id=$property_id";
+
 		wp_enqueue_script( // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.MissingVersion
 			'google_gtagjs',
-			'https://www.googletagmanager.com/gtag/js?id=' . esc_attr( $property_id ),
+			$gtag_src,
```

Comment:
Escaping the `$property_id` isn't necessary as the whole URL runs through `esc_url` internally.